birdspotter/birdspotter/dataio/views.py

```diff
@@ -1,10 +1,16 @@
-from django.contrib.auth.decorators import login_required
-from django.shortcuts import render, redirect
+from django.contrib.auth.decorators import login_required, user_passes_test
```

Comment:
user_passes_test is unused according to the linter - is this because there is missing code, or does it just need to be removed?